test/verify/check-system-info

```diff
@@ -756,6 +756,13 @@ fi
         if not m.image.startswith("rhel"):
             self.skipTest("insights-client is only on RHEL")
 
+        # insights-client might get started during boot and might then
+        # run concurrently with our explicit "insights-client
+        # --register" below.  insights-client is not designed to be
+        # run concurrently and there is no protection against it,
+        # apparently.  So let's prevent that.
+        m.execute("systemctl disable --now insights-client")
```

Comment:
Eww.. I guess that's the "I want to file a bug report" you mentioned on IRC? If you did, would be nice to add a `# HACK` reference. Especially as it's enabled OOTB, and using the insights CLI should be a supported use case.

Reply:
Yep. I find it hard to care, but I filed a bug now: https://bugzilla.redhat.com/show_bug.cgi?id=2175652